locale/pt-br/texts.po

```diff
@@ -57,6 +378,302 @@ msgstr "Saldo Disponível"
 msgid "Locked"
 msgstr "Bloqueado"
 
+#: src/screens/PaymentRequestDetail.js:69
+#, javascript-format
+msgid "You've just received **${ amount } ${ symbol }**"
+msgstr "Você recebeu **${ amount } ${ symbol }**"
+
+#: src/screens/PaymentRequestDetail.js:86
+msgid "PAYMENT REQUEST"
+msgstr "REQUISIÇÃO DE PAGAMENTO"
+
+#: src/screens/PaymentRequestDetail.js:104
+msgid "Token"
+msgstr "Token"
+
+#: src/screens/PaymentRequestDetail.js:116 src/screens/Receive.js:29
+msgid "My Address"
+msgstr "Meu endereço"
+
+#: src/screens/PaymentRequestDetail.js:120
+msgid "Status"
+msgstr "Status"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Received"
+msgstr "Recebido"
+
+#: src/screens/PaymentRequestDetail.js:121
+msgid "Waiting confirmation"
+msgstr "Aguardando confirmação"
+
 #: src/screens/PinScreen.js:157
 msgid "Incorrect PIN Code. Try again."
 msgstr "PIN incorreto. Tente novamente."
+
+#: src/screens/PinScreen.js:53
+msgid "Enter your PIN Code "
+msgstr "Entre seu PIN"
+
+#: src/screens/PinScreen.js:54
+msgid "Unlock Hathor Wallet"
+msgstr "Desbloqueie sua Hathor Wallet"
+
+#: src/screens/PinScreen.js:169
+msgid "Cancel"
+msgstr "Cancelar"
+
+#: src/screens/PinScreen.js:172 src/screens/Settings.js:128
+msgid "Reset wallet"
+msgstr "Resetar a wallet"
+
+#. *
+#.      * index {number} Selected index of the tab bar
+#.      
+#.  eslint thinks routes is not used, but TabView uses it
+#.  eslint-disable-next-line react/no-unused-state
+#: src/screens/Receive.js:30
+msgid "Payment Request"
+msgstr "Requisição de Pagamento"
+
+#: src/screens/Receive.js:105
+msgid "RECEIVE"
+msgstr "RECEBER"
+
+#: src/screens/RegisterToken.js:31 src/screens/SendScanQRCode.js:67
+msgid "Manual info"
+msgstr "Digitar"
```

Comment:
I only know this makes sense because I know where this is from. Maybe a comment or context can help.

Reply:
Done.